test/remote/gateways/remote_litle_test.rb

```diff
@@ -207,13 +213,13 @@ def test_successful_purchase_with_3ds_fields
   end
 
   def test_successful_purchase_with_apple_pay
-    assert response = @gateway.purchase(10010, @decrypted_apple_pay)
+    assert response = @gateway.purchase(1000, @decrypted_apple_pay, @options)
     assert_success response
     assert_equal 'Approved', response.message
   end
 
   def test_successful_purchase_with_android_pay
-    assert response = @gateway.purchase(10000, @decrypted_android_pay)
+    assert response = @gateway.purchase(10000, @decrypted_android_pay, @options)
```

Comment:
❓  Question:
Why the value on the test above is 1000 and this one is 10000, is a "magic" value ?

Reply:
Just an update in the value because of the multiple trying in the process of correcting tests, in this case is not a magic value